coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -113,6 +127,14 @@ def _parse_documentation_with_symbols(self,
             section is a named tuple of either ``Description``, ``Parameter``,
             ``ExceptionValue`` or ``ReturnValue``.
         """
+        def _reference(line, ref_identifiers):
```

Comment:
Is there a more suitable place for this helper function?

I also plan to remove some of the algorithmic redundancy in the other checks by introducing another helper function  - We may then place both of these together.

Reply:
Place them at the module scope with `_` prefix, so they can be tested separately